.github/workflows/tests.yml

```
@@ -134,11 +134,17 @@ jobs:
   trial-olddeps:
     # Note: sqlite only; no postgres
     if: ${{ !cancelled() && !failure() }} # Allow previous steps to be skipped, but not fail
-    needs: linting-done
+    # needs: linting-done
```

```suggestion
    needs: linting-done
```

(just making sure we don't forget before hitting merge)